index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Some meta tags -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portofolio Website</title>
+    <!-- Link to the Font-Awesome Font -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" />
+    <!-- Link to Roboto Font-->
+    <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
+    <!-- Styles -->
+    <!-- Custom Styles -->
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <nav class="nav-container">
+        <ul class="navbar-list">
+            <li class="name1">John Deo</li>
+            <li class="hamburger-menu"><i class="fa fa-bars"></i></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] It would be better if you wrap your `nav` inside a `<header>` for semantic reasons and also because the `<header>` tag in HTML is used to define the header for a document or a section as it contains the information related to the title and heading of the related content.
- [ ] Kindly ensure that your logo is a link tag. Use the `<a>` link tag 😊
- [ ] OPTIONAL: For the logo name, you can replace it with your name as it's your personal portfolio
- [ ] Instead of using a `<ul>`, you can just wrap each in its `div` because they are not related to a certain list.